charts/nutanix-csi-storage/README.md

```diff
@@ -128,7 +128,7 @@ The following table lists the configurable parameters of the Nutanix-CSI chart a
 | `prismEndPoint`               | Prism Element (PE) cluster Virtual IP Address or fully qualified domain name (FQDN)         |                        |
 | `username`                    | Username of a Prism Element (PE) cluster admin (if created)                                 |                        |
 | `password`                    | Password for the Prism Element (PE) cluster admin (if created)                              |                        |
-| `secretName`                  | Secret name that stores Prism Element (PE) cluster credentials                              | `ntnx-secret`          |
+| `peSecretName`                | Secret name that stores Prism Element (PE) cluster credentials                              | `ntnx-secret`          |
```

Comment:
why are we changing this?

Reply:
I other places, this secret is referred as peSecretName - https://github.com/subodh01/helm/pull/10/files#diff-345a7a29336e8c320e1bf3c09298d5e0a8132f2f1b31d89f9bf319acad4d0ad3R82
Also, since we have pcSecretName, we can also have peSecretName to avoid any confusion.